src-frontend/app/templates/login.hbs

```diff
@@ -15,9 +15,10 @@
             {{input type="password" value=password class="form-control"}}
           </div>
         </div>
-        <div class="form-group">
+        <div class="form-inline">
           <div class="col-sm-offset-2 col-sm-10">
             <button type="submit" class="btn btn-default">Log in</button>
+            <button type="submit" {{action 'signup'}} class="btn btn-default">Sign up</button>
```

Comment:
You can just use the `{{link-to}}` helper instead of using the `signup` action to call `transitionToRoute` :star: